theano/gpuarray/basic_ops.py

```diff
@@ -968,16 +970,22 @@ class GpuAllocEmpty(HideC, AllocEmpty):
     Allocate uninitialized memory on the GPU.
 
     """
-    __props__ = ('dtype', 'context_name')
+    __props__ = ('dtype', 'context_name', 'typecode')
```

Comment:
@tfjgeorge It may be because of this line. As `typecode` is just a class property (Python class property), we don't need to set it as a Theano op property. So remove `'typecode'` from `__props__`.